src/Symfony/Bundle/FrameworkBundle/Command/CachePoolClearCommand.php

```diff
@@ -0,0 +1,83 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bundle\FrameworkBundle\Command;
+
+use Psr\Cache\CacheItemPoolInterface;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+use Symfony\Component\HttpKernel\CacheClearer\Psr6CacheClearer;
+
+/**
+ * Clear cache pools.
+ *
+ * @author Nicolas Grekas <p@tchwork.com>
+ */
+class CachePoolClearCommand extends ContainerAwareCommand
```

Comment:
@unkind I think that this is indeed a good conversation to have, but we should have it globally, not just for this PR (or any other ones). I'm not sold yet on adding final as it's a hard stop on inheritance and extensibility. I like the current way as we have a very clear BC promise (and we can probably add some specific notes about build-in commands, controllers, ...) but if people want to go wild, they can (knowing they are on their own).